src/workerd/api/basics.c++

```diff
@@ -407,6 +407,55 @@ jsg::Ref<AbortSignal> AbortSignal::timeout(jsg::Lock& js, double delay) {
   return kj::mv(signal);
 }
 
+jsg::Ref<AbortSignal> AbortSignal::any(
+    jsg::Lock& js,
+    kj::Array<jsg::Ref<AbortSignal>> signals,
+    const jsg::TypeHandler<EventTarget::HandlerFunction>& handler) {
+  // If nothing was passed in, we can just return a signal that never aborts.
+  if (signals.size() == 0) {
+    return jsg::alloc<AbortSignal>(nullptr, nullptr, AbortSignal::Flag::NEVER_ABORTS);
+  }
+
+  // Let's check to see if any of the signals are already aborted. If it is, we can
+  // optimize here by skipping the event handler registration.
+  for (auto& sig : signals) {
+    if (sig->getAborted()) return AbortSignal::abort(js, sig->getReason(js));
```

Comment:
Nitpick: don't know if our style guide disallows it, but I think it's worthwhile to be explicit to avoid any subtle issues down the line.

```suggestion
    if (sig->getAborted()) { return AbortSignal::abort(js, sig->getReason(js)) };
```

Reply:
Just in case it saves time, the semi-colon's are awry in the suggestion.